torchci/lib/github.ts

```diff
@@ -8,25 +7,11 @@ export async function getOctokit(
   owner: string,
   repo: string
 ): Promise<Octokit> {
-  let privateKey = process.env.PRIVATE_KEY as string;
+  let privateKey = process.env.PYTORCH_HUD_TOKEN as string;
   privateKey = Buffer.from(privateKey, "base64").toString();
 
-  const app = new App({
```

Comment:
jeeze, we just had this dead code lying around?

Reply:
it was used later on for getting the app installation, but now its dead :))